test/integration/src/main/java/io/pravega/test/integration/HDFSIntegrationTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.integration;
+
+import io.pravega.segmentstore.server.store.ServiceBuilder;
+import io.pravega.segmentstore.server.store.ServiceBuilderConfig;
+import io.pravega.segmentstore.storage.chunklayer.ChunkedSegmentStorageConfig;
+import io.pravega.segmentstore.storage.impl.bookkeeper.BookKeeperConfig;
+import io.pravega.segmentstore.storage.impl.bookkeeper.BookKeeperLogFactory;
+import io.pravega.storage.hdfs.HDFSClusterHelpers;
+import io.pravega.storage.hdfs.HDFSSimpleStorageFactory;
+import io.pravega.storage.hdfs.HDFSStorageConfig;
+import io.pravega.storage.hdfs.HDFSStorageFactory;
+import lombok.val;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.junit.After;
+import org.junit.Before;
+
+/**
+ * End-to-end tests for SegmentStore, with integrated Storage and DurableDataLog.
+ */
+public class HDFSIntegrationTest extends BookKeeperIntegrationTestBase {
```

Comment:
this needs to be in `hdfs` gradle project. I don't think should have integration tests outside a binding gradle project.